packages/playwright/src/runner/runner.ts

```diff
@@ -70,7 +81,28 @@ export class Runner {
       report.projects.push({
         name: project.project.name,
         testDir: project.project.testDir,
-        use: { testIdAttribute: project.project.use.testIdAttribute },
+        use: {
```

Comment:
I don't think we should send this information back and forth, it looks like this should rather be a part of the test server.